generators/rust/sdk/versions.yml

```diff
@@ -1,5 +1,19 @@
 # yaml-language-server: $schema=../../../fern-versions-yml.schema.json
 
+- version: 0.7.3
```

Comment:
hey @tjb9dc, 

Just a mention, if we observe here on the version bumps, have need to add 2 version bumps up, because of the upgrade of `ir-sdk` from `v59` --> `v61`

Want to double-check, this kind of scenario is fine?